### DISCUSSION TITLE
Adding RTSP stream to Scrypted

### DISCUSSION THREAD

**Discussion by ctestst** [roleoroleo/yi-hack-Allwinner#336] [General]
I'm having issues on adding the camera (yi home BFUS fw 0.2.7) stream on Scrypted, looks like the rtsp stream is not stable, it works for some hours then it stops. Anyone has been able to successfully do it?

**Comment by pidybi** [comment]
BTW is it possible to get h264 stream with a web request? (y25ga 9FUS fw 0.3.1)

**Reply by roleoroleo** [reply]
> I'm having issues on adding the camera (yi home BFUS fw 0.2.7) stream on Scrypted, looks like the rtsp stream is not stable, it works for some hours then it stops. Anyone has been able to successfully do it?

Try alternative rtsp server.

> BTW is it possible to get h264 stream with a web request? (y25ga 9FUS fw 0.3.7)

With wich protocol?

**Reply by pidybi** [reply]
> With wich protocol?

just simple http like others cameras
eg Sannce i41cg works with
`<img class="w-100" alt="cam" src="http://192.168.0.27:5441/videostream.cgi?loginuse=my_login&loginpas=my_pass">`

edit: the software of YI doesn't seem to support it :(

**Reply by roleoroleo** [reply]
I think it's a mjpeg stream.
I'm not able to change stream codec from h264 to mjpeg because this part is managed by Yi software.

**Comment by pidybi** [comment]
ok, I see, so my "dirty" solution is:

`<img id="yi" class="w-100" alt="" src="http://192.168.0.49:8080/cgi-bin/snapshot.sh?res=low&watermark=yes">`

and JS

`<script>`
		`setInterval(function() {`
			`var myImageElement = document.getElementById('yi');`
			`myImageElement.src = 'http://192.168.0.49:8080/cgi-bin/snapshot.sh?res=low&watermark=yes&rand=' + Math.random();`
		`}, 500);`
	`</script>`

:)
